### ISSUE TITLE
Error when retrieving an existing Linode instance object

### ISSUE THREAD

**Issue by justinmayer** [linode/python-linode-api#52]
I can create Linode instances via v4 API Python library, but I cannot subsequently retrieve the created instance object.

### Environment

```
$ python --version
Python 3.6.2

$ pip list | grep linode
linode-api (4.1.4b0)
```

### Steps to Reproduce

```
$ python

> import linode
> client = linode.LinodeClient($MY_TOKEN)
> linode, pw = client.linode.create_instance('g5-nanode-1', 'us-west-1a', distribution='linode/ubuntu16.04lts')
> print(linode.id, linode.status)
1234567 running

> l = linode.Linode(client, linode.id)
```

### Expected Result

The `l` variable would refer to the queried Linode instance object.

### Observed Result

Error output:

```
/lib/python3.6/site-packages/linode/objects/base.py in __init__(self, client, id, json)
     84             self._set(getattr(type(self), 'id_attribute'), id)
     85
---> 86         self._populate(json)
     87
     88     def __getattribute__(self, name):

/lib/python3.6/site-packages/linode/objects/linode/linode.py in _populate(self, json)
    141     def _populate(self, json):
    142         # fixes ipv4 and ipv6 attribute of json to make base._populate work
--> 143         if 'ipv4' in json and 'address' in json['ipv4']:
    144             json['ipv4']['id'] = json['ipv4']['address']
    145         if 'ipv6' in json and isinstance(json['ipv6'], list):

TypeError: argument of type 'NoneType' is not iterable
```

### Additional Notes

The docs are not consistent as to whether the Linode instance ID should be passed as an integer or as a string (see: [1](https://developers.linode.com/v4/reference/endpoints/linode/instances/$id), [2](https://developers.linode.com/v4/libraries/python/linode)). But I have tried passing both and the same error results in both cases.

Is there any additional information I can provide regarding this problem?

**Comment by kirys** [comment]
I've the same issue

**Comment by Dorthu** [comment]
This is fixed in v4.1.4b1
